e2e/testcafe-devextreme/tests/pivotGrid/virtualScrolling_T1210807.ts

```diff
@@ -0,0 +1,194 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
```

Comment:
This e2e test's PivotGrid configuration contains a lot of redundant data.
And the data set is too big and contains a lot of unnecessary fields.

The original issue [T1210807](https://supportcenter.devexpress.com/ticket/details/t1210807/pivotgrid-row-fields-overlap-data-fields-if-datafieldarea-is-set-to-row-and-virtual) reproduction case can be simplified:
[Link to the simplified example](https://codepen.io/williamvinogradov/pen/eYaRaxN?editors=0010)

So, let's take a simplified example as a basis
To make this e2e test easiest to support/understand in the future :)